content/panorama/layout/custom_game/common/patreon/patreon.js

```diff
@@ -253,6 +254,30 @@ SubscribeToNetTableKey('game_state', 'patreon_bonuses', function (data) {
 	boots.Enable( !!status.bootsEnabled )
 });
 
+SubscribeToNetTableKey('player_info', 'steam_ids', function (data) {
+	UpdatePaymentTargetList(data)
+});
+
+function UpdatePaymentTargetList(data) {
+	var dropdown_parent = $('#PaymentWindowUserSelectorContainer');
+	var donation_target_dropdown = $.CreatePanel('DropDown', dropdown_parent, 'PaymentWindowDropDown');
+	var layout_string = '<root><DropDown style="margin-left: 5px;" oninputsubmit="updatePaymentWindow()" >';
+
+	for(var id = 0; id <= 40; id++) {
+		if (Players.IsValidPlayerID(id)) {
+			layout_string = layout_string + '<Label text="' + Players.GetPlayerName(id) +'" id="PatreonOption' + id + '" onmouseover="UpdatePaymentTarget(' + id + ')" />';
```

Comment:
ES6 template strings work pretty nice here
```suggestion
			layout_string += `<Label text="${Players.GetPlayerName(id)}" id="PatreonOption${id}" onmouseover="UpdatePaymentTarget(${id})" />`;
```

Reply:
very nice